frontend/src/models/models.js

```diff
@@ -89,6 +90,41 @@ module.exports = app => app.component('models', {
     this.status = 'loaded';
   },
   methods: {
+    clickFilter(path) {
+       // Initialize searchFieldsMap if empty
+      if (!this.searchText) {
+        this.searchFieldsMap = {};
```

Comment:
Make searchFieldsMap a local variable, no need to track searchFieldsMap on the component state because you don't use searchFieldsMap in the HTML